docs/modules/migrate/pages/dmt-run.adoc

```diff
@@ -51,3 +51,15 @@ To run the migration, complete the following steps:
 You can use the DMT `status` command to track the migration. 
 
 For further information on the available DMT commands, see the xref:migrate:dmt-command-reference.adoc[DMT Command Reference].
+
+=== Unsuccessful Migration
+
+The DMT migrates the data structures one-by-one.
+
+If the migration of a data structure is not successful, the DMT does the following:
+
+* Retains any data structures that have been successfully migrated already
+* Reverts the unsuccessful data structure migration
+* Cancels the migration of all remaining data structures
+
+In this situation, you can restart the migration [information to be confirmed].
```

Comment:
yes they can restart the migration. currently the way to do that is just restarting the migration with the same command. existing data structures in the target cluster will be skipped.